input/pagecontent/reference.md

```diff
@@ -1705,11 +1705,11 @@ When indented rules are combined with [soft indexing](#array-paths-using-soft-in
 
 </div>
 
-#### AddElement Rules
+#### Add Element Rules
```

Comment:
Please double check to make sure any references use (#add-element-rules) not (#addelement-rules)